src/provider/rsa_padding.c

```diff
@@ -524,116 +812,127 @@ int ibmca_rsa_check_oaep_mgf1_padding(const struct ibmca_prov_ctx *provctx,
                     inlen, outsize, EVP_MD_get0_name(oaep_md),
                     EVP_MD_get0_name(mgf1_md), label_len);
 
-    oaep_md_len = EVP_MD_get_size(oaep_md);
-    if (oaep_md_len <= 0) {
+    mdlen = EVP_MD_get_size(oaep_md);
+    if (mdlen <= 0) {
```

Comment:
unsigned mdlen is never < 0, no need to check

Reply:
in deed. removed the check.